ps_landing_page/models/hr_employee_landing_page.py

```diff
@@ -260,29 +306,42 @@ def action_view_analytic_tree(self):
         self.ensure_one()
         ir_model_data = self.env["ir.model.data"]
         tree_res = ir_model_data.get_object_reference(
-            "ps_timesheet_invoicing", "view_ps_time_line_tree"
+            "ps_landing_page", "view_ps_time_line_landing_page_tree"
         )
         tree_id = tree_res and tree_res[1] or False
 
         user_id = self.env.user.id
+        # self.env.cr.execute(
+        #     """
+        #     SELECT
+        #         id
+        #         FROM ps_time_line
+        #         WHERE user_id = %s
+        #           AND ot = true
+        #     UNION
+        #     SELECT
+        #         ps_time_line.id
+        #         FROM ps_time_line
+        #         JOIN project_project
+        #         ON ps_time_line.project_id=project_project.id
+        #         AND overtime
+        #         WHERE ps_time_line.user_id = %s
+        #         AND ps_time_line.state != 'draft'
+        #     """,
+        #     (user_id, user_id),
+        # )
+
         self.env.cr.execute(
             """
             SELECT
                 id
                 FROM ps_time_line
                 WHERE user_id = %s
-                  AND ot = true
-            UNION
-            SELECT
-                ps_time_line.id
-                FROM ps_time_line
-                JOIN project_project
-                ON ps_time_line.project_id=project_project.id
-                AND overtime
-                WHERE ps_time_line.user_id = %s
-                AND ps_time_line.state != 'draft'
+                AND state != 'new'
```

Comment:
ps.time.line#state doesn't have a value `new`, so this will also show unconfirmed lines - is that intended?

Reply:
Again my bad! didn't check the attribute state values defined for ps_time_line, I was trying to show entries from ps_time_line in a draft state as well, might I not pass this in where clause.
I would recheck and add my changes.

Reply:
nothing to apologize for, we're here to improve stuff together :-)

Reply:
@hbrunn @Stephanvandenberg 
While pulling changes from tosccolors:14.0 to kumarinie:14.0 there was a conflict, I did reset --hard to kumarinie:14.0 so automatically PR is closed. However, I'll add fixes based on feedback.